Tella/Scenes/Settings/ViewModel/UwaziServerViewModel.swift

```diff
@@ -314,5 +323,39 @@ class UwaziServerViewModel: ObservableObject {
         self.username = server.username ?? ""
         self.password = server.password ?? ""
     }
+    
+    
+    fileprivate func addPrivateInstanceServer() {
```

Comment:
maybe we need a significant function name like  `getServerSettings` or `getServerName`

Reply:
but we are using this function to add the private instance. The thing is that we need to fetch the settings first to get the name of the server.  So naming this getServerSettings would be inaccurate because we are also adding a server.